kafka/librdkafka/import.sh

```diff
@@ -0,0 +1,88 @@
+#!/bin/bash
+#
+#
+# Import a new version of librdkafka based on a librdkafka static bundle.
+# This will create a separate branch, import librdkafka, make a commit,
+# and then ask you to push the branch to github, have it reviewed,
+# and then later merged (NOT squashed or rebased).
+# Having a merge per import allows future shallow clones to skip and ignore
+# older imports, hopefully reducing the amount of git history data 'go get'
+# needs to download.
```

Comment:
seems like shallow clone isn't going to be a default or option with go get any time soon: https://github.com/golang/go/issues/13078#issuecomment-152062051

(no actionable feedback here, more noting what i'm reading)

Reply:
Right, I think there are Go plans to use snapshots rather than clones in the future, which would be equivalent of `--depth 1` minus the .git index (so even smaller).

`git clone --depth 1 .. `  is still an option for people that highly care of import sizes.

I've done some explorative testing and these are the resulting numbers:
 * A standard librdkafka three platform build import is ~8MB per platform, a total of 24MB.
 * This consumes 24MB in the .gitindex and 24MB in the checkout itself = 48MB
 * Compressing the index (which is done by github and by manual `git gc`) brings the index size down to about 8MB
 * Importing 3 more releases, to a total of 4 releases, brings the index size up to 42MB, uncompressed.
 * Compression that index again brings it down to 8MB again. This makes sense since these releases are nearly identical.
 * Performing a shallow clone (`--depth 1`) gives a clone that consumes 17MB: 8MB in the git index, 7MB in the checkout